src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/Tool/ProvisioningToolOptions.cs

```diff
@@ -12,13 +13,28 @@ public class ProvisioningToolOptions : IDeveloperCredentialsOptions
         /// <summary>
         /// Path to csproj file 
         /// </summary>
-        public string? ProjectCsProjPath { get; set; }
+        public string? ProjectFilePath{ get; set; }
 
         /// <summary>
         /// Path to appsettings.json file
         /// </summary>
         public string? AppSettingsFilePath { get; set; }
 
+        ///<summary>
+        /// Web redirect URIs.
+        ///</summary>
+        public IList<string> RedirectUris { get; set; } = new List<string>();
+
+        ///<summary>
+        /// enable id tokens to be issued by the authZ endpoint
+        ///</summary>
+        public bool? EnableIdToken { get; set; }
```

Comment:
This is mixing level of abstractions!!!
I'd be interested in understanding the why? and the design you have in mind.

- For RedirectUris, I can imagine that you'd want to use it for deployment? (please confirm).
- But what is IdToken and Access token. Do you mean the implicit flow idtokens and access tokens? why would you do that? that's a security impact. This should be inferred from the type of app and guided by the metadata?

Reply:
Yes, we want to use RedirectUris for deployment and update scenarios.
For implicit flow, these should be removed. We weren't sure initially if VS would need that or if it can be inferred by metadata.

Reply:
Will be removed in an upcoming PR.Will be removed in an upcoming PR.